pyrobuf/parse_proto.py

```diff
@@ -24,6 +24,7 @@ class Parser(object):
         ('MAP_FIELD', r'map<([A-Za-z][0-9A-Za-z_]+),\s*([A-Za-z][0-9A-Za-z_]+)>\s+([A-Za-z][0-9A-Za-z_]*)\s*=\s*(\d+)'),
         ('DEFAULT', r'default\s*='),
         ('PACKED', r'packed\s*=\s*(true|false)'),
+        ('RESERVED', r'reserved\s*([0-9A-Za-z_]+\s*,?\s*)+;'),
```

Comment:
Nit, but think there has to be at least one space after `reserved`, so maybe `reserved\s+` instead of `reserved\s*`.